### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by WilcoSchoolderman** [fiduswriter/Simple-DataTables#406] [Q&A] [answered]
Is it posible to add language: { url: "/static/i18n/nl-NL.json" }, like in dataTables.net with an i18n json file?

**Comment by johanneswilm** [comment, marked as answer]
Hey @WilcoSchoolderman ,
it works differently here. You can just specify different strings and in that way control the text that is seen. In some languages you may want to use the formal or the informal language (du/Sie in German). So there are many different ways that localization can be done for a website, so this is the way that gives the widest-most way of configuring it. In Fidus Writer we use it in a localized environment and specify the strings like this:

https://github.com/fiduswriter/fiduswriter/blob/6c287755bfd4d8681d4abbac3c2aa04f46372085/fiduswriter/bibliography/static/js/modules/bibliography/overview/index.js#L88-L91

```js
new DataTable(tableEl, {
    labels: {
       noRows: gettext("No sources registered"),
       noResults: gettext("No sources found") // Message shown when there are no search results
   }
})
```

The use of `gettext` there means that it will pick up the strings from our translation files for the entire website.
